api/api/spec/changes.md

```diff
@@ -1,383 +1,528 @@
 ## General
-* Changed parameter **status** type *string* to *array*
 * Date type use a standard format ISO-8601 defined by date-time format.
-* Changed parameter **agent_id** type *integer* to *string* with minLength=3
-* Changed all return parameters **agent_id** type *integer* to *string*
 * Deleted all return parameters **path**, new API don't show any absolute path in responses.
-* `error` field has been removed. Now error status is shown in HTTP status code (400 for client error and 500 for server error)
-* `data` is never showing a human readable message. To be consistent, it will only contain an object or list of objects. In case
-a human readable message is shown, the new field `message` will be used instead.
 * Changed search negation from `!` to `-`.
 * Changed nested fields from `a_b` to `a.b`
 * Changed parameter **query** to allow reserved characters.
+* The endpoint's responses has been changed accordingly to the new RBAC standard. See spec schema responses carefully.
+* The responses no longer will have `items` and `totalitems` fields, instead most responses will have the following structure:
+```
+{
+  "data": {
+    "affected_items": [],
+    "total_affected_items": 0,
+    "total_failed_items": 0,
+    "failed_items": [],
+    "message": ""
+  }
+}
+```
+* In responses, `msg` key is now moved to new `message` key
+* In responses, `filter` key is now moved to new `filters` key
```

Comment:
Remove this line, I haven`t found any response that uses this key anymore.

Reply:
Done